stix_shifter_modules/cisco_secure_email/configuration/lang_en.json

```diff
@@ -0,0 +1,38 @@
+{
+    "connection": {
+        "host": {
+            "label": "Management IP address or hostname",
+            "description": "Specify the IP address or hostname of the data source"
+        },
+        "port": {
+            "label": "Host port",
+            "description": "Set the port number that is associated with the hostname or IP address"
+        },
+        "help": {
+            "label": "Need additional help?",
+            "description": "More details on the data source setting can be found in the specified link"
+        },
+        "selfSignedCert": {
+            "label": "Cisco Secure Email HTTPS connection certificate",
+            "description": "Use self-signed Security Sockets Layer (SSL) or CA certificate for HTTPS services"
+        },
+         "options": {
+             "result_limit": {
+                 "label": "Result size limit",
+                 "description": "The maximum number of entries or objects that are returned by search query.  Valid input range is {{min}} to {{max}}."
```

Comment:
do you see {{min}} and {{max}} showing up in the UI description?

Reply:
In the UI, it shows from the common configuration that values are not taken from the connector configuration. 

The below description is shown in CP4S
"The maximum number of entries or objects that are returned by search query. Valid input range is 1 to 10000000."